website/content/docs/examples/bootstrap-k0s.md

```diff
@@ -71,10 +72,28 @@ bctl apply -f k0s-in-aws-with-tf.yaml
 bctl update -f k0s-in-aws-with-tf.yaml
 ```
 
+5. View the status of the cluster's Kubernetes pods:
+```
+kubectl get pods --all-namespaces
+NAMESPACE          NAME                                                     READY   STATUS              RESTARTS   AGE
+boundless-system   boundless-operator-controller-manager-677b86bdc4-rtjwb   1/2     Running             0          25s
+boundless-system   helm-controller-79cc59c76b-vsr2v                         1/1     Running             0          5s
+default            helm-install-nginx-mj2qt                                 0/1     ContainerCreating   0          3s
+kube-system        coredns-878bb57ff-d4j99                                  1/1     Running             0          40s
+kube-system        konnectivity-agent-jkz62                                 1/1     Running             0          39s
+kube-system        kube-proxy-22rxj                                         1/1     Running             0          39s
+kube-system        kube-router-mrbks                                        1/1     Running             0          39s
+kube-system        metrics-server-7f86dff975-gs26h                          0/1     Running             0          40s
+```
+
 #### Accessing the cluster
 
 The example app addon can now be accessed through the `http://<controller-node-ip>:6443` URL.
 
+##### Managing AWS VMs
+
+Navigate the AWS's EC2 page and select the appropriate region. Additional details regarding the AWS VMs can be found there.
```

Comment:
"Navigate the AWS's" -> "Navigate to the AWS EC2"
"select the appropriate region" -> "select the appropriate region from the dropdown in the top right of the page"

Is there a way to do this using terraform or the AWS CLI that we could include here?

Reply:
> Is there a way to do this using terraform or the AWS CLI that we could include here?

Something like this, perhaps?

```bash
aws ec2 describe-regions --query 'Regions[*].RegionName' --output table
```

Example:

```bash
❯ aws ec2 describe-regions --query 'Regions[*].RegionName' --output table

--------------------
|  DescribeRegions |
+------------------+
|  ap-south-1      |
|  eu-north-1      |
|  eu-west-3       |
|  eu-west-2       |
|  eu-west-1       |
|  ap-northeast-3  |
|  ap-northeast-2  |
|  ap-northeast-1  |
|  ca-central-1    |
|  sa-east-1       |
|  ap-southeast-1  |
|  ap-southeast-2  |
|  eu-central-1    |
|  us-east-1       |
|  us-east-2       |
|  us-west-1       |
|  us-west-2       |
+------------------+
```

If you were thinking of something more terraform-esque, it's not yet possible to have terraform dynamically work out a list of values to be validated in `variables.tf`.

Reply:
Sticking with terraform would be nice but understandable if it's not possible. I was thinking a way to list the instances from the CLI. Something like 
```
aws ec2 describe-instances
```

It might also be better to put this in the `#### Create virtual machines on AWS`

Reply:
> I was thinking a way to list the instances from the CLI. Something like
> 
> ```
> aws ec2 describe-instances
> ```

Ah :) yeah that's definitely possible, likely involving [this](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/instances) and making it part of the output. Probably better off in a separate PR, though.